security/cmd/node_agent_k8s/nodeagent.yaml

```diff
@@ -19,8 +19,10 @@ spec:
           mountPath: /host/driver
       containers:
       - name: nodeagent
-        image: gcr.io/istio-testing/nodeagent:latest
+        image: gcr.io/jianfeih-test/node-agent-k8s:latest
```

Comment:
To still use istio-testing project's image, I need permission to push image to this project. @wattli do you know how to get that?

Reply:
@ayj , do you have the permission?

Reply:
incfly, you should have permission now. Let me know if there is any problem.

Reply:
Thanks, changing to the istio-testing now.